examples/sample_measurement/measurement.py

```diff
@@ -38,6 +43,12 @@ class Color(Enum):
 @sample_measurement_service.configuration(
     "Enum In", nims.DataType.Enum, Color.BLUE, enum_type=Color
 )
+@sample_measurement_service.configuration(
+    "Protobuf Enum In",
+    nims.DataType.Enum,
+    color_pb2.ProtobufColor.BLACK,
+    enum_type=color_pb2.ProtobufColor,  # type: ignore
```

Comment:
`# type: ignore` should not be necessary here unless you are testing with the previous release of ni-measurementlink-service, in which case protobuf enums don't work at all.

Reply:
Ok, I see what's going on.

The runtime type is `google.protobuf.internal.enum_type_wrapper.EnumTypeWrapper`, but the type used in `color_pb2.pyi` is `google.protobuf.internal.enum_type_wrapper._EnumTypeWrapper[_V]`:
```
class _ProtobufColor:
    ValueType = typing.NewType("ValueType", builtins.int)
    V: typing_extensions.TypeAlias = ValueType

class _ProtobufColorEnumTypeWrapper(google.protobuf.internal.enum_type_wrapper._EnumTypeWrapper[_ProtobufColor.ValueType], builtins.type):
    DESCRIPTOR: google.protobuf.descriptor.EnumDescriptor
    NONE: _ProtobufColor.ValueType  # 0
    PINK: _ProtobufColor.ValueType  # 1
    WHITE: _ProtobufColor.ValueType  # 2
    BLACK: _ProtobufColor.ValueType  # 3

class ProtobufColor(_ProtobufColor, metaclass=_ProtobufColorEnumTypeWrapper):
    """---------------------------------------------------------------------
    ---------------------------------------------------------------------
    """
```

In the type stubs, `EnumTypeWrapper` inherits from `_EnumTypeWrapper[int]`. `_EnumTypeWrapper[_V]` does not inherit from `EnumTypeWrapper`.
https://github.com/python/typeshed/blob/main/stubs/protobuf/google/protobuf/internal/enum_type_wrapper.pyi
```
_V = TypeVar("_V", bound=int)

# Expose a generic version so that those using mypy-protobuf
# can get autogenerated NewType wrapper around the int values
class _EnumTypeWrapper(Generic[_V]):
    DESCRIPTOR: EnumDescriptor
    def __init__(self, enum_type: EnumDescriptor) -> None: ...
    def Name(self, number: _V) -> str: ...
    def Value(self, name: str | bytes) -> _V: ...
    def keys(self) -> list[str]: ...
    def values(self) -> list[_V]: ...
    def items(self) -> list[tuple[str, _V]]: ...

class EnumTypeWrapper(_EnumTypeWrapper[int]): ...
```

This seems to work:
```
diff --git a/ni_measurementlink_service/measurement/service.py b/ni_measurementlink_service/measurement/service.py
index 3e70dc4..7ba372a 100644
--- a/ni_measurementlink_service/measurement/service.py
+++ b/ni_measurementlink_service/measurement/service.py
@@ -40,9 +40,9 @@ from ni_measurementlink_service.measurement.info import (
 from ni_measurementlink_service.session_management import PinMapContext

 if TYPE_CHECKING:
-    from google.protobuf.internal.enum_type_wrapper import EnumTypeWrapper
+    from google.protobuf.internal.enum_type_wrapper import EnumTypeWrapper, _EnumTypeWrapper, _V

-    SupportedEnumType = Union[Type[Enum], EnumTypeWrapper]
+    SupportedEnumType = Union[Type[Enum], EnumTypeWrapper, _EnumTypeWrapper[_V]]

```

Reply:
Correction: I think this is sufficient. I don't think we need `EnumTypeWrapper` and `_V`.
```
SupportedEnumType = Union[Type[Enum], _EnumTypeWrapper]
```

Reply:
Switched to `SupportedEnumType = Union[Type[Enum], _EnumTypeWrapper]` as suggested. Thanks for all the help on these issues.